task3/src/config.js

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const databaseConfig = {
+    database: process.env.DATABASE_NAME,
+    username: process.env.DATABASE_USERNAME,
+    password: process.env.DATABASE_PASSWORD,
+    host: process.env.DATABASE_HOST,
+    port: process.env.DATABASE_PORT,
+    dialect: process.env.DATABASE_DIALECT
+};
+
+export default {
+    port: process.env.PORT,
+    databaseConfig
```

Comment:
Why do you need to export databaseConfig & then to reexport it via default?

Reply:
I did it because I couldn't destructure databaseConfig in my import when I needed it, because it's default export and they cannot be destructured in import statement. As I'm willing to use process.env.VAR_NAME directly in my modules, it won't be a problem.

Reply:
Use 2 named exports if you have some problems with default